.github/docker/Dockerfile.debian

```diff
@@ -1,3 +1,3 @@
-FROM debian:bullseye-20230612
+FROM debian:bookworm-20230612
```

Comment:
Hehe, I was looking at this the other day, and was waiting to see what the LTS schedule is.

https://wiki.debian.org/LTS

But, I trust you know better this world. And if you say it's stable, then so be it.